beacontools/scanner.py

```diff
@@ -141,14 +136,14 @@ def set_scan_parameters(self, scan_type=ScanType.ACTIVE, interval_ms=10, window_
         interval_fractions, window_fractions = int(interval_fractions), int(window_fractions)
 
         scan_parameter_pkg = struct.pack(
-            ">BHHBB",
+            "<BHHBB",
```

Comment:
why did you change this?

Reply:
Because Bluetooth uses little endian everywhere (again, see commit messages)

How did you get the idea that this should be big endian?

Reply:
I never questioned it because I thought it works™. But now I realize changing the interval and window values has no impact if they are big endian. Thanks for the hint!